tools/utils/code_change_tools.ts

```diff
@@ -1,17 +1,24 @@
-import {PORT, APP_DEST} from '../config';
+import {PORT, APP_DEST, APP_BASE, DIST_DIR} from '../config';
 import * as browserSync from 'browser-sync';
 
 let runServer = () => {
+  let baseDir = APP_DEST;
   let routes:any = {
-    [`/${APP_DEST}`]: APP_DEST,
-    '/node_modules': 'node_modules'
+    [`${APP_BASE}${APP_DEST}`]: APP_DEST,
+    [`${APP_BASE}node_modules`]: 'node_modules',
   };
+
+  if (APP_BASE !== '/') {
+    routes[`${APP_BASE}`] = APP_DEST;
+    baseDir = `${DIST_DIR}/empty/`;
```

Comment:
@ludohenin: removing it shouldn't stop anything working, but removing it does make things work that shouldn't. 

If it's not there then the project files are served at both / and /APP_BASE/, which is why the connect-history-api-fallback was working when it shouldn't of been. i.e it was loading /index.html instead of /APP_BASE/index.html. 

It's a problem if you want your dev environment to work the same way as production.